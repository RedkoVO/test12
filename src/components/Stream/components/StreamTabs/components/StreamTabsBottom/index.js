import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import styles from './styles'

const StreamTabsBottom = ({ classes }) => (
  <div className="streamTabsBottom">
    <div className="streamTabsNav">
      <a href="#" className="streamTabsNavItem liveNow">
        LIVE NOW <span className="streamTabsWatch">2</span>
      </a>
      <a href="#" className="streamTabsNavItem allGames">
        ALL GAMES <span className="streamTabsWatch">2</span>
      </a>
      <a href="#" className="streamTabsNavItem  dota2">
        DOTA 2<span className="streamTabsWatch">2</span>
      </a>
      <a href="#" className="streamTabsNavItem starcraft2">
        STARCRAFT 2 <span className="streamTabsWatch">2</span>
      </a>
      <a href="#" className="streamTabsNavItem overwatch">
        OVERWATCH <span className="streamTabsWatch">2</span>
      </a>
      <a href="#" className="streamTabsNavItem rainbowSix">
        RAINBOW SIX <span className="streamTabsWatch">2</span>
      </a>
      <a href="#" className="streamTabsNavItem csGo streamTabsNavActive ">
        CS:GO<span className="streamTabsWatch">2</span>
      </a>
    </div>

    <div className="streamTabsLive">
      <div className="streamLiveItem">
        <div className="streamLiveItemDate itemDateLive">LIVE</div>
        <div className="streamLiveItemTeam teamNRG">
          <div className="teamName">NRG</div>
          <div className="teamCountry teamFrance">FRANCE</div>
        </div>
        <div className="streamLiveItemScore">6.14</div>
        <div className="streamLiveItemVs">VS</div>
        <div className="streamLiveItemScore">1.25</div>
        <div className="streamLiveItemTeam teamLDLC">
          <div className="teamName">LDLC</div>
          <div className="teamCountry teamSpain">SPAIN</div>
        </div>
        <div className="streamLiveItemEvent">EVENT</div>
        <div className="streamLiveItemLink" />
      </div>
      <div className="streamLiveItem">
        <div className="streamLiveItemDate ">
          12:30 PM <br />
          <span className="dateDay">7/12/2018</span>
        </div>
        <div className="streamLiveItemTeam teamNRG">
          <div className="teamName">NRG</div>
          <div className="teamCountry teamFrance">FRANCE</div>
        </div>
        <div className="streamLiveItemScore">6.14</div>
        <div className="streamLiveItemVs">VS</div>
        <div className="streamLiveItemScore">1.25</div>
        <div className="streamLiveItemTeam teamLDLC">
          <div className="teamName">LDLC</div>
          <div className="teamCountry teamSpain">SPAIN</div>
        </div>
        <div className="streamLiveItemEvent">EVENT</div>
        <div className="streamLiveItemLink" />
      </div>
      <div className="streamLiveItem">
        <div className="streamLiveItemDate itemDateLive">LIVE</div>
        <div className="streamLiveItemTeam teamNRG">
          <div className="teamName">NRG</div>
          <div className="teamCountry teamFrance">FRANCE</div>
        </div>
        <div className="streamLiveItemScore">6.14</div>
        <div className="streamLiveItemVs">VS</div>
        <div className="streamLiveItemScore">1.25</div>
        <div className="streamLiveItemTeam teamLDLC">
          <div className="teamName">LDLC</div>
          <div className="teamCountry teamSpain">SPAIN</div>
        </div>
        <div className="streamLiveItemEvent">EVENT</div>
        <div className="streamLiveItemLink" />
      </div>
      <div className="streamLiveItem">
        <div className="streamLiveItemDate ">
          12:30 PM <br />
          <span className="dateDay">7/12/2018</span>
        </div>
        <div className="streamLiveItemTeam teamNRG">
          <div className="teamName">NRG</div>
          <div className="teamCountry teamFrance">FRANCE</div>
        </div>
        <div className="streamLiveItemScore">6.14</div>
        <div className="streamLiveItemVs">VS</div>
        <div className="streamLiveItemScore">1.25</div>
        <div className="streamLiveItemTeam teamLDLC">
          <div className="teamName">LDLC</div>
          <div className="teamCountry teamSpain">SPAIN</div>
        </div>
        <div className="streamLiveItemEvent">EVENT</div>
        <div className="streamLiveItemLink" />
      </div>
      <div className="streamLiveItem">
        <div className="streamLiveItemDate itemDateLive">LIVE</div>
        <div className="streamLiveItemTeam teamNRG">
          <div className="teamName">NRG</div>
          <div className="teamCountry teamFrance">FRANCE</div>
        </div>
        <div className="streamLiveItemScore">6.14</div>
        <div className="streamLiveItemVs">VS</div>
        <div className="streamLiveItemScore">1.25</div>
        <div className="streamLiveItemTeam teamLDLC">
          <div className="teamName">LDLC</div>
          <div className="teamCountry teamSpain">SPAIN</div>
        </div>
        <div className="streamLiveItemEvent">EVENT</div>
        <div className="streamLiveItemLink" />
      </div>
      <div className="streamLiveItem">
        <div className="streamLiveItemDate ">
          12:30 PM <br />
          <span className="dateDay">7/12/2018</span>
        </div>
        <div className="streamLiveItemTeam teamNRG">
          <div className="teamName">NRG</div>
          <div className="teamCountry teamFrance">FRANCE</div>
        </div>
        <div className="streamLiveItemScore">6.14</div>
        <div className="streamLiveItemVs">VS</div>
        <div className="streamLiveItemScore">1.25</div>
        <div className="streamLiveItemTeam teamLDLC">
          <div className="teamName">LDLC</div>
          <div className="teamCountry teamSpain">SPAIN</div>
        </div>
        <div className="streamLiveItemEvent">EVENT</div>
        <div className="streamLiveItemLink" />
      </div>
      <div className="streamLiveItem">
        <div className="streamLiveItemDate itemDateLive">LIVE</div>
        <div className="streamLiveItemTeam teamNRG">
          <div className="teamName">NRG</div>
          <div className="teamCountry teamFrance">FRANCE</div>
        </div>
        <div className="streamLiveItemScore">6.14</div>
        <div className="streamLiveItemVs">VS</div>
        <div className="streamLiveItemScore">1.25</div>
        <div className="streamLiveItemTeam teamLDLC">
          <div className="teamName">LDLC</div>
          <div className="teamCountry teamSpain">SPAIN</div>
        </div>
        <div className="streamLiveItemEvent">EVENT</div>
        <div className="streamLiveItemLink" />
      </div>
      <div className="streamLiveItem">
        <div className="streamLiveItemDate ">
          12:30 PM <br />
          <span className="dateDay">7/12/2018</span>
        </div>
        <div className="streamLiveItemTeam teamNRG">
          <div className="teamName">NRG</div>
          <div className="teamCountry teamFrance">FRANCE</div>
        </div>
        <div className="streamLiveItemScore">6.14</div>
        <div className="streamLiveItemVs">VS</div>
        <div className="streamLiveItemScore">1.25</div>
        <div className="streamLiveItemTeam teamLDLC">
          <div className="teamName">LDLC</div>
          <div className="teamCountry teamSpain">SPAIN</div>
        </div>
        <div className="streamLiveItemEvent">EVENT</div>
        <div className="streamLiveItemLink" />
      </div>
      <div className="streamLiveItem">
        <div className="streamLiveItemDate itemDateLive">LIVE</div>
        <div className="streamLiveItemTeam teamNRG">
          <div className="teamName">NRG</div>
          <div className="teamCountry teamFrance">FRANCE</div>
        </div>
        <div className="streamLiveItemScore">6.14</div>
        <div className="streamLiveItemVs">VS</div>
        <div className="streamLiveItemScore">1.25</div>
        <div className="streamLiveItemTeam teamLDLC">
          <div className="teamName">LDLC</div>
          <div className="teamCountry teamSpain">SPAIN</div>
        </div>
        <div className="streamLiveItemEvent">EVENT</div>
        <div className="streamLiveItemLink" />
      </div>
      <div className="streamLiveItem">
        <div className="streamLiveItemDate ">
          12:30 PM <br />
          <span className="dateDay">7/12/2018</span>
        </div>
        <div className="streamLiveItemTeam teamNRG">
          <div className="teamName">NRG</div>
          <div className="teamCountry teamFrance">FRANCE</div>
        </div>
        <div className="streamLiveItemScore">6.14</div>
        <div className="streamLiveItemVs">VS</div>
        <div className="streamLiveItemScore">1.25</div>
        <div className="streamLiveItemTeam teamLDLC">
          <div className="teamName">LDLC</div>
          <div className="teamCountry teamSpain">SPAIN</div>
        </div>
        <div className="streamLiveItemEvent">EVENT</div>
        <div className="streamLiveItemLink" />
      </div>
    </div>
  </div>
)

StreamTabsBottom.propTypes = {
  classes: PropTypes.object
}

export default withStyles(styles)(StreamTabsBottom)
