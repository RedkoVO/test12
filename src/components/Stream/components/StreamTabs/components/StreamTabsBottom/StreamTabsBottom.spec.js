import * as React from 'react'
import renderer from 'react-test-renderer'

import StreamTabsBottom from './'

it('renders correctly', () => {
  const tree = renderer.create(<StreamTabsBottom />).toJSON()
  expect(tree).toMatchSnapshot()
})
