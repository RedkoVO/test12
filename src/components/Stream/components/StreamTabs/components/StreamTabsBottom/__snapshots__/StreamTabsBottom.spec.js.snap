// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="StreamTabsBottom-root-1"
>
  <div
    className="StreamTabsNav-root-2"
  >
    <div
      className="StreamTabsNav-streamTabsNavItem-3 StreamTabsNav-liveNow-4"
    >
      LIVE NOW 
      <span
        className={undefined}
      >
        2
      </span>
    </div>
    <div
      className="StreamTabsNav-streamTabsNavItem-3 StreamTabsNav-allGames-5"
    >
      ALL GAMES 
      <span
        className={undefined}
      >
        2
      </span>
    </div>
    <div
      className="StreamTabsNav-streamTabsNavItem-3 StreamTabsNav-dota2-6"
    >
      DOTA 2
      <span
        className={undefined}
      >
        2
      </span>
    </div>
    <div
      className="StreamTabsNav-streamTabsNavItem-3 StreamTabsNav-starcraft2-7"
    >
      STARCRAFT 2 
      <span
        className={undefined}
      >
        2
      </span>
    </div>
    <div
      className="StreamTabsNav-streamTabsNavItem-3 StreamTabsNav-overwatch-8"
    >
      OVERWATCH 
      <span
        className={undefined}
      >
        2
      </span>
    </div>
    <div
      className="StreamTabsNav-streamTabsNavItem-3 StreamTabsNav-rainbowSix-9"
    >
      RAINBOW SIX 
      <span
        className={undefined}
      >
        2
      </span>
    </div>
    <div
      className="StreamTabsNav-streamTabsNavItem-3 StreamTabsNav-csGo-11 StreamTabsNav-streamTabsNavActive-10"
    >
      CS:GO
      <span
        className={undefined}
      >
        2
      </span>
    </div>
  </div>
  <div
    className="StreamLive-root-12"
  >
    <div
      className={undefined}
    >
      <div
        className=""
      >
        Live
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          NRG
        </div>
        <div
          className=""
        >
          France
        </div>
      </div>
      <div
        className={undefined}
      >
        6.14
      </div>
      <div
        className={undefined}
      >
        VS
      </div>
      <div
        className={undefined}
      >
        1.25
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          LDLC
        </div>
        <div
          className=""
        >
          Spain
        </div>
      </div>
      <div
        className={undefined}
      >
        EVENT
      </div>
      <div
        className={undefined}
      />
    </div>
    <div
      className={undefined}
    >
      <div
        className=""
      >
        12:30 PM
         
        <br />
        <span
          className="dateDay"
        >
          7/12/2018
        </span>
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          NRG
        </div>
        <div
          className=""
        >
          France
        </div>
      </div>
      <div
        className={undefined}
      >
        6.14
      </div>
      <div
        className={undefined}
      >
        VS
      </div>
      <div
        className={undefined}
      >
        1.25
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          LDLC
        </div>
        <div
          className=""
        >
          Spain
        </div>
      </div>
      <div
        className={undefined}
      >
        EVENT
      </div>
      <div
        className={undefined}
      />
    </div>
    <div
      className={undefined}
    >
      <div
        className=""
      >
        Live
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          NRG
        </div>
        <div
          className=""
        >
          France
        </div>
      </div>
      <div
        className={undefined}
      >
        6.14
      </div>
      <div
        className={undefined}
      >
        VS
      </div>
      <div
        className={undefined}
      >
        1.25
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          LDLC
        </div>
        <div
          className=""
        >
          Spain
        </div>
      </div>
      <div
        className={undefined}
      >
        EVENT
      </div>
      <div
        className={undefined}
      />
    </div>
    <div
      className={undefined}
    >
      <div
        className=""
      >
        12:30 PM
         
        <br />
        <span
          className="dateDay"
        >
          7/12/2018
        </span>
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          NRG
        </div>
        <div
          className=""
        >
          France
        </div>
      </div>
      <div
        className={undefined}
      >
        6.14
      </div>
      <div
        className={undefined}
      >
        VS
      </div>
      <div
        className={undefined}
      >
        1.25
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          LDLC
        </div>
        <div
          className=""
        >
          Spain
        </div>
      </div>
      <div
        className={undefined}
      >
        EVENT
      </div>
      <div
        className={undefined}
      />
    </div>
    <div
      className={undefined}
    >
      <div
        className=""
      >
        Live
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          NRG
        </div>
        <div
          className=""
        >
          France
        </div>
      </div>
      <div
        className={undefined}
      >
        6.14
      </div>
      <div
        className={undefined}
      >
        VS
      </div>
      <div
        className={undefined}
      >
        1.25
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          LDLC
        </div>
        <div
          className=""
        >
          Spain
        </div>
      </div>
      <div
        className={undefined}
      >
        EVENT
      </div>
      <div
        className={undefined}
      />
    </div>
    <div
      className={undefined}
    >
      <div
        className=""
      >
        12:30 PM
         
        <br />
        <span
          className="dateDay"
        >
          7/12/2018
        </span>
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          NRG
        </div>
        <div
          className=""
        >
          France
        </div>
      </div>
      <div
        className={undefined}
      >
        6.14
      </div>
      <div
        className={undefined}
      >
        VS
      </div>
      <div
        className={undefined}
      >
        1.25
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          LDLC
        </div>
        <div
          className=""
        >
          Spain
        </div>
      </div>
      <div
        className={undefined}
      >
        EVENT
      </div>
      <div
        className={undefined}
      />
    </div>
    <div
      className={undefined}
    >
      <div
        className=""
      >
        Live
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          NRG
        </div>
        <div
          className=""
        >
          France
        </div>
      </div>
      <div
        className={undefined}
      >
        6.14
      </div>
      <div
        className={undefined}
      >
        VS
      </div>
      <div
        className={undefined}
      >
        1.25
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          LDLC
        </div>
        <div
          className=""
        >
          Spain
        </div>
      </div>
      <div
        className={undefined}
      >
        EVENT
      </div>
      <div
        className={undefined}
      />
    </div>
    <div
      className={undefined}
    >
      <div
        className=""
      >
        12:30 PM
         
        <br />
        <span
          className="dateDay"
        >
          7/12/2018
        </span>
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          NRG
        </div>
        <div
          className=""
        >
          France
        </div>
      </div>
      <div
        className={undefined}
      >
        6.14
      </div>
      <div
        className={undefined}
      >
        VS
      </div>
      <div
        className={undefined}
      >
        1.25
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          LDLC
        </div>
        <div
          className=""
        >
          Spain
        </div>
      </div>
      <div
        className={undefined}
      >
        EVENT
      </div>
      <div
        className={undefined}
      />
    </div>
    <div
      className={undefined}
    >
      <div
        className=""
      >
        Live
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          NRG
        </div>
        <div
          className=""
        >
          France
        </div>
      </div>
      <div
        className={undefined}
      >
        6.14
      </div>
      <div
        className={undefined}
      >
        VS
      </div>
      <div
        className={undefined}
      >
        1.25
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          LDLC
        </div>
        <div
          className=""
        >
          Spain
        </div>
      </div>
      <div
        className={undefined}
      >
        EVENT
      </div>
      <div
        className={undefined}
      />
    </div>
    <div
      className={undefined}
    >
      <div
        className=""
      >
        12:30 PM
         
        <br />
        <span
          className="dateDay"
        >
          7/12/2018
        </span>
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          NRG
        </div>
        <div
          className=""
        >
          France
        </div>
      </div>
      <div
        className={undefined}
      >
        6.14
      </div>
      <div
        className={undefined}
      >
        VS
      </div>
      <div
        className={undefined}
      >
        1.25
      </div>
      <div
        className=""
      >
        <div
          className={undefined}
        >
          LDLC
        </div>
        <div
          className=""
        >
          Spain
        </div>
      </div>
      <div
        className={undefined}
      >
        EVENT
      </div>
      <div
        className={undefined}
      />
    </div>
  </div>
</div>
`;
